shacl12-core/index.html

```diff
@@ -1734,6 +1739,166 @@ <h4>Objects-of targets (sh:targetObjectsOf)</h4>
 							because it is the <a>object</a> of a <a>triple</a> that has <code>ex:knows</code> as its <a>predicate</a>.
 						</p>
 					</section>
+					
+					<section id="targetWhere">
+						<h4>Where Targets (sh:targetWhere)</h4>
+						<p class="syntax">
+							A <dfn>where target</dfn> is specified with the <code>sh:targetWhere</code> predicate.
+							<span data-syntax-rule="targetWhere-node">Each value of <code>sh:targetWhere</code> in a shape is a <a>well-formed</a> <a>shape</a>.</span>
+						</p>
+						<div class="def">
+							<div class="def-header">TEXTUAL DEFINITION</div>
+							If <code>s</code> is a shape in a shapes graph <code>SG</code> and <code>s</code> has <a>value</a> <code>w</code> for
+							<code>sh:targetWhere</code> in <code>SG</code> then the set of <a>nodes</a> in a data graph
+							<code>DG</code> that <a>conform</a> to <code>w</code> is a <a>target</a> from <code>DG</code> for <code>s</code> in <code>SG</code>.
+						</div>
+						<p><em>The remainder of this section is informative.</em></p>
+						<aside class="example">
+							<div class="shapes-graph">
+								<div class="turtle">
+ex:PersonShape
+    a sh:NodeShape ;
+    sh:targetClass ex:Person ;
+    sh:property [
+        sh:path ex:age ;
+        sh:datatype xsd:integer ;
+        sh:maxCount 1 ;
+    ].
+
+ex:AdultShape
```

Comment:
```suggestion
ex:AdultPerson
```
In the explanation `ex:AdultPerson` is used instead of `ex:AdultShape`.